src/mainwindow.h

```diff
@@ -689,6 +689,13 @@ private slots:
     QActionGroup *languageGroup;
     QActionGroup *spellBackendGroup;
     bool _brokenTagNoteLinksRemoved = false;
+    const QIcon tagIcon = QIcon::fromTheme(
```

Comment:
can you please use `_` as prefix for global variables

Reply:
Yes, sorry I am really forgetful 😬